### PR TITLE
[TR] Add Project Loved

### PR DESCRIPTION
**Pull request by ubinici**
Add Turkish translations for `Project_Loved` article

### CODE REVIEW

File: wiki/Project_Loved/tr.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Project Loved
+
+**Project Loved** beatmapleri [Sevilen](/wiki/Beatmaps#loved) kategorisine kazandıran bir oylama plaformudur. Bu proje birincil olarak [Project Loved Takımı](/wiki/People/The_Team/Project_Loved_Team) tarafından yürütülür.
+
+## Bir beatmapin Sevilenlere eklenmesi
+
+Beatmapler, bir aday gösterme ve oylama süreci ile Sevilenlere eklenir.
+
+**Kaptanlar** olarak bilinen Project Loved Takımı üyeleri spesifik bir oyun modu için beatmap setlerini düzenli olarak aday gösterirler. Neredeyse her zaman için, bu setler osu! topluluğundan gelen gönderilerden seçilir. Mevcut Sevilen gönderi formlarına ve listelerine aşağıdaki linklerden ulaşılabilir:
```
Comment (3dsosu) [inline wiki/Project_Loved/tr.md:9]:
```suggestion
**Kaptanlar** olarak bilinen Project Loved Takımı üyeleri spesifik bir oyun modu için beatmap setlerini düzenli olarak aday gösterirler. Neredeyse her zaman, bu setler osu! topluluğundan gelen gönderilerden seçilir. Mevcut Sevilen gönderi formlarına ve listelerine aşağıdaki linklerden ulaşılabilir:
```

File: wiki/Project_Loved/tr.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+*Not: Eğer Google Formlara erişiminiz kısıtlıysa, bir beatmapi sizin adınıza göndermesi için herhangi bir [Project Loved Takım üyesiyle](/wiki/People/The_Team/Project_Loved_Team#team-members) iletişime geçebilirsiniz.*
+
+Aday gösterilen map setleri daha sonrasında, topluluk üyelerinin map setinin Sevilen olmasıni isteyip istemediklerini oylayabilecekleri [Project Loved forumunda](https://osu.ppy.sh/community/forums/120) paylaşılır. Oylamaların bitiminde "Evet" oyları belirli bir yüzdeliğe ulaşan mapler Sevilen kategorisine eklenir.
+
+### Uygunluk
+
+Beatmaplerin Sevilen kategorisine aday gösterilebilmesi için karşılaması gereken minimal birtakım kriterler bulunmaktadır. Kaptanlar bir beatmap setini aşağıdaki kriterleri sağlıyorsa göz önünde bulundurur:
+
+- [Nitelikli](/wiki/Beatmaps#qualified), [Dereceli](/wiki/Beatmaps#ranked) or [Onaylı](/wiki/Beatmaps#approved) kategorilerinde olmamalı.
```
Comment (3dsosu) [inline wiki/Project_Loved/tr.md:26]:
```suggestion
- [Nitelikli](/wiki/Beatmaps#qualified), [Dereceli](/wiki/Beatmaps#ranked) ya da [Onaylı](/wiki/Beatmaps#approved) kategorilerinde olmamalı.
```

File: wiki/Project_Loved/tr.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# Project Loved
+
+**Project Loved** beatmapleri [Sevilen](/wiki/Beatmaps#loved) kategorisine kazandıran bir oylama plaformudur. Bu proje birincil olarak [Project Loved Takımı](/wiki/People/The_Team/Project_Loved_Team) tarafından yürütülür.
```
Comment (Lxran) [inline wiki/Project_Loved/tr.md:3]:
```suggestion
**Project Loved** beatmapleri [Sevilen](/wiki/Beatmaps#loved) kategorisine kazandıran oylama plaformudur. Bu proje birincil olarak [Project Loved Takımı](/wiki/People/The_Team/Project_Loved_Team) tarafından yürütülür.
```

File: wiki/Project_Loved/tr.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+- [Nitelikli](/wiki/Beatmaps#qualified), [Dereceli](/wiki/Beatmaps#ranked) ya da [Onaylı](/wiki/Beatmaps#approved) kategorilerinde olmamalı.
+- Yaratıcısı setin Sevilen olmasını reddetmemeli.
+- Yaratıcısı banlı ya da kısıtlı olmamalı.
+- En az bir zorluğun geçilebilir olmalı.
+- En az 30 favoriye sahip olmalı (sadece osu!standard mapleri için geçerlidir)
+
+Ancak, neredeyse tüm [Beklemede ve Yapım aşamasındaki](/wiki/Beatmaps#work-in-progress-and-pending) maplerin bu kriterlere uymasına rağmen, nispeten daha seçici aday gösterme ve oylama gereksinimlerinden dolayı yalnızca birkaçı Sevilen kategorisine eklenmesi için oylanır.
+
+Sevilen kategorisine giren beatmapler birtakım kuralları takip etmek zorundadır:
```
Comment (Lxran) [inline wiki/Project_Loved/tr.md:34]:
```suggestion
Sevilen kategorisine giren beatmapler birtakım ek kuralları takip etmek zorundadır:
```

**Review by Lxran — CHANGES_REQUESTED** [review]
Minor edits to better reflect the source text. Good work.

**Review by TicClick — APPROVED** [review]

**Pull request merged**